packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -102,6 +107,8 @@ class RenderingEngine {
       this.offscreenMultiRenderWindow =
         vtkOffscreenMultiRenderWindow.newInstance();
       this.offScreenCanvasContainer = document.createElement('div');
+      this.offScreenCanvasContainer.style.width = `${MAX_WIDTH}px`;
```

Comment:
This immediately consumes the full size - it seems like a bad idea on limited devices.  What about tesselating the current total screen area as a height/width?  In theory that allows for any viewport displayed on screen non-overlapping, and won't consume more memory than the current screen size.